app/models/aggregated_tag_mapping.rb

```diff
@@ -0,0 +1,28 @@
+class AggregatedTagMapping
+  class Link
+    attr_accessor :link_title, :link_content_id, :link_type
+
+    def initialize(link_title:, link_content_id:, link_type:)
```

Comment:
If you use `include ActiveModel::Model` you don't need to create an accessor, the `attr_accessor` is enough. More details [here](http://api.rubyonrails.org/classes/ActiveModel/Model.html)